build-aux/m4/ax_boost_base.m4

```diff
@@ -72,7 +72,7 @@ AC_ARG_WITH([boost-libdir],
 )
 
 if test "x$want_boost" = "xyes"; then
-    boost_lib_version_req=ifelse([$1], ,1.20.0,$1)
+    boost_lib_version_req=ifelse([$1], ,1.47.0,$1)
```

Comment:
concept ACK - though personally I'd prefer passing in this version as a parameter from our configure.ac, if possible, instead of changing this upstream file